lib/src/test/kotlin/network/account/AccountTest.kt

```diff
@@ -787,19 +775,18 @@ class AccountTest {
         Thread.sleep(120000)
 
         val address = deployer.findContractAddress(deployment).send()
+        println(address)
         assertNotEquals(Felt.ZERO, address)
     }
 
     @Test
     fun `test udc deploy with constructor`() {
         assumeTrue(NetworkConfig.isTestEnabled(requiresGas = true))
 
-        assumeFalse(network == Network.GOERLI_INTEGRATION)
+        assumeFalse(network == Network.SEPOLIA_INTEGRATION)
         val classHash = when (network) {
-            Network.GOERLI_TESTNET -> Felt.fromHex("0x31de86764e5a6694939a87321dad5769d427790147a4ee96497ba21102c8af9")
             Network.SEPOLIA_TESTNET -> Felt.fromHex("0x8448a68b5ea1affc45e3fd4b8b480ea36a51dc34e337a16d2567d32d0c6f8a")
             Network.SEPOLIA_INTEGRATION -> Felt.fromHex("0x31de86764e5a6694939a87321dad5769d427790147a4ee96497ba21102c8af9")
-            else -> throw NotImplementedError("Test is not yet supported for this network $network")
```

Comment:
I think this should be left as it was, because otherwise the when loop is not exhaustive

Reply:
It's exhaustive, we removed `Network.GOERLI_TESTNET` and `.GOERLI_INTEGRATION`